src/components/result-ui/summary.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import {LinkElement} from "../link";
+
+const flattenObject = (obj, previous = '') => {
+  const flattened = {};
+  Object.entries(obj).forEach(([property, value]) => {
+    const key = `${previous.length ? `${previous}.` : ''}${property}`;
+    let assign;
+    if (value && typeof value === 'object') {
+      assign = flattenObject(value, key);
+    }  else {
+      assign = {[key]: value};
+    }
+    Object.assign(flattened, assign);
+  });
+  return flattened;
+};
+
+const FieldRow = ({ fieldPath, fieldValue, focused, focus, defocus }) => {
+  return (
+    <>
+      <div className={"result-row-path"}>
+        {focused
+          ? <a className={"result-row-field-focus-link"} onClick={focus} title="Hide other fields"><span className={"arrow"}>&uArr;</span>{fieldPath}</a>
+          : <a className={"result-row-field-focus-link"} onClick={defocus} title="Show all fields"><span className={"arrow"}>&dArr;</span>{fieldPath}</a>
+        }
+      </div>
+      <div className={"result-row-value"}>{JSON.stringify(fieldValue)}</div>
+    </>
+  );
+};
+
+const Summary = ({data, focusPath, defaultZoom}) => {
+  const [zoom, setZoom] = useState(defaultZoom);
+  const [focus, setFocus] = useState(focusPath);
```

Comment:
These should probably reset when you switch from resource to resource, if you have a specific attribute focused then change to view a resource that does not have that attribute it just shows every row as empty with an "expand" button